inc/switchtec/switchtec.h

```diff
@@ -610,6 +610,23 @@ static inline const char *switchtec_variant_str(struct switchtec_dev *dev)
 	return str;
 }
 
+/**
+ * @brief Return the phase string for a phase id.
+ */
+static inline const char* phase_id_to_string(enum switchtec_boot_phase phase_id)
```

Comment:
Might be a good idea to rename this with a switchtec_prefix, or some such.

Reply:
Updated, thanks for the review!